proposals/lambda-method-group-defaults.md

```diff
@@ -69,41 +81,54 @@ app.MapPost("/todos/{id}", todoHandler);
 ```
 
 ## Current Behavior
-Currently, when a user implements a lambda with a default value, the compiler raises an error stating that default values are not allowed. 
+Currently, when a user implements a lambda with a default or `params` parameter, the compiler raises an error stating that it's not allowed. 
```

Comment:
>raises an error stating that it's not allowed.  [](http://example.com/codeflow?start=94&length=47)

Since there are examples below, perhaps just "reports an error."